sdk/core/src/_defaultConfig.ts

```diff
@@ -6,7 +6,8 @@ export const DEV_REGISTRY_URL = 'https://affinity-registry.dev.affinity-project.
 export const DEV_ISSUER_URL = 'https://affinity-issuer.dev.affinity-project.org'
 export const DEV_VERIFIER_URL = 'https://affinity-verifier.dev.affinity-project.org'
 export const DEV_KEY_STORAGE_URL = 'https://affinity-wallet-backend.dev.affinity-project.org'
-export const DEV_VAULT_URL = 'https://bloom-vault.dev.affinity-project.org'
+export const DEV_BLOOM_VAULT_URL = 'https://bloom-vault.dev.affinity-project.org'
+export const DEV_VAULT_URL = 'https://affinidi-vault.dev.affinity-project.org'
```

Comment:
Better rename it to `DEV_AFFINIDI_VAULT_URL` so that it won't be misleading (so that developers who relied on `DEV_VAULT_URL` will at least learn about backwards incompatibility straight away at compile time rather than silently getting the wrong URL)

Reply:
My thoughts was about that bloom-vault becomes deprecated and only affinidi-vault left as single credential storage, but yeah that could be renamed until bloom-vault will be removed to avoid issues you described.